app/assets/javascripts/iframe-saver.coffee

```diff
@@ -9,11 +17,13 @@ class IFrameSaver
   constructor: ($iframe, $data_div, $delete_button) ->
     @$iframe = $iframe
     @$delete_button = $delete_button
+    @save_state = $data_div.data('save-state');
     @interactive_run_state_url = $data_div.data('interactive-run-state-url') # get and put our data here.
     @collaborator_urls = $data_div.data('collaborator-urls')
     @auth_provider = $data_div.data('authprovider') # through which provider did the current user log in
     @user_email = $data_div.data('user-email')
     @logged_in = $data_div.data('loggedin') # true/false - is the current session associated with a user
+    @authoredState = getAuthoredState($data_div) # state / configuration provided during authoring
```

Comment:
Yes I mean within iframe-saver.coffee.  And possibly in the interactive_run_helper.rb